Containerfile.bpfman.openshift

```diff
@@ -0,0 +1,52 @@
+# We do not use --platform feature to auto fill this ARG because of incompatibility between podman and docker
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1194 AS bpfman-build
+ARG DNF_CMD="microdnf"
+
+ARG BUILDPLATFORM
+
+# The following ARGs are set internally by docker/build-push-action in github actions
+ARG TARGETARCH=amd64
+ARG TARGETOS=linux
+ARG TARGETPLATFORM=linux/amd64
+
+RUN echo "TARGETOS=${TARGETOS}  TARGETARCH=${TARGETARCH}  BUILDPLATFORM=${BUILDPLATFORM}  TARGETPLATFORM=${TARGETPLATFORM}"
+
+WORKDIR /usr/src/bpfman
+COPY ./ /usr/src/bpfman
+
+RUN ${DNF_CMD} update -y \
+    && ${DNF_CMD} install -y rust cargo openssl-devel \
```

Comment:
Do you know what version of cargo we're getting?

Reply:
```console
% podman run -it --rm registry.access.redhat.com/ubi9/ubi-minimal:9.4-1194 /bin/bash
[root@c964ec992e9a /]# microdnf update && microdnf repoquery cargo

(microdnf:8): librhsm-WARNING **: 09:25:15.384: Found 0 entitlement certificates

(microdnf:8): librhsm-WARNING **: 09:25:15.386: Found 0 entitlement certificates
Downloading metadata...
Downloading metadata...
Downloading metadata...
Package                                                                                                                                        Repository                                              Size
Upgrading:
 curl-minimal-7.76.1-29.el9_4.1.x86_64                                                                                                         ubi-9-baseos-rpms                                   131.8 kB
  replacing curl-minimal-7.76.1-29.el9_4.x86_64
 libcurl-minimal-7.76.1-29.el9_4.1.x86_64                                                                                                      ubi-9-baseos-rpms                                   233.7 kB
   replacing libcurl-minimal-7.76.1-29.el9_4.x86_64
Transaction Summary:
 Installing:        0 packages
 Reinstalling:      0 packages
 Upgrading:         2 packages
 Obsoleting:        0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Is this ok [y/N]: y
Downloading packages...
Running transaction test...
Updating: libcurl-minimal;7.76.1-29.el9_4.1;x86_64;ubi-9-baseos-rpms
Updating: curl-minimal;7.76.1-29.el9_4.1;x86_64;ubi-9-baseos-rpms
Cleanup: curl-minimal;7.76.1-29.el9_4;x86_64;installed
Cleanup: libcurl-minimal;7.76.1-29.el9_4;x86_64;installed
Complete.

(microdnf:66): librhsm-WARNING **: 09:25:18.082: Found 0 entitlement certificates

(microdnf:66): librhsm-WARNING **: 09:25:18.084: Found 0 entitlement certificates

cargo-1.75.0-1.el9.x86_64
```